app/assets/javascripts/cascade/plugins/flexslider.js

```diff
@@ -3,4 +3,9 @@
  * Copyright 2012 WooThemes
  * Contributing Author: Tyler Smith
  */
+
+/* CHANGES:
+* Added descriptive title attributes to paging navigation - sharissis-brei
```

Comment:
👍  Yes, this is what I had in mind. Can you date it and explain the why behind change, like this (I'd do it myself but I don't have access to your repo):

`2016-09-13: Added descriptive title attributes to paging navigation for ADA compliance. - sharissis-brei`

Also, please include this TODO below it:

```
TODO(tatwell): Before any future changes, update to latest version. Right now we're at 
2.2.0. Current version (https://github.com/woothemes/FlexSlider) is 2.6.3. May need to 
redo ADA-compliance changes we made to this version. 
```

We're trying to get a little more rigorous with our JS dev practices over here and code comments and maintainability are a focus. Thanks!